### ISSUE TITLE
Example project

### ISSUE THREAD

**Issue by willf80** [toukea/android-IstatSms#1]
Hello,
Your library is great but there are not example to implement this. Can i help you to contribute ?

**Comment by toukea** [comment]
Yes, in fact. i have only write a litle Readme with use case to guide developper durring implementation:
https://github.com/toukea/android-IstatSms/blob/master/README.md.
Ofcourse a sample would be a great idea. do you have some plane?
